stan/math/opencl/rev/matrix_power.hpp

```diff
@@ -34,7 +34,6 @@ inline var_value<matrix_cl<double>> matrix_power(const var_value<T>& M,
   if (M.size() == 0)
     return M;
 
-  size_t N = M.rows();
```

Comment:
This seems like at least one place where it's obvious the variable should be used (replacing the 3 calls below) rather than removed. Not a huge issue though, since this function is always pretty cheap?

Reply:
Agreed!